editor/editor_node.cpp

```diff
@@ -4172,8 +4199,8 @@ void EditorNode::_scene_tab_script_edited(int p_tab) {
 		inspector_dock->edit_resource(script);
 }
 
-void EditorNode::_scene_tab_closed(int p_tab) {
-	current_option = SCENE_TAB_CLOSE;
+void EditorNode::_scene_tab_closed(int p_tab, int option) {
```

Comment:
I think this connection signature may need updating.

https://github.com/godotengine/godot/blob/6c8f15083618b0b8dff738bb2257744c0acd7cd1/editor/editor_node.cpp#L5441

In certain conditions that I haven't isolated yet, I can get the following error, and be unable to close the tab.

![image](https://user-images.githubusercontent.com/13004169/56723840-6cb08e00-6752-11e9-8059-39074c8ddb63.png)

Reply:
Thanks for testing, I managed to reproduce it and pushed a fix.

Reply:
No problem, thanks for lending your expertise to add this feature, it will be a productivity bump for many.  X)